safe-svg.php

```diff
@@ -500,12 +500,24 @@ protected function svg_dimensions( $svg ) {
                  *
                  * @return {bool} If we should use the width & height attributes first or not.
                  */
-                if ( (bool) apply_filters( 'safe_svg_use_width_height_attributes', false, $svg ) ) {
-                    $width  = $attr_width;
-                    $height = $attr_height;
+                $use_width_height = (bool) apply_filters( 'safe_svg_use_width_height_attributes', false, $svg );
+
+                if ( $use_width_height ) {
+                    if ( isset( $attr_width, $attr_height ) ) {
+                        $width  = $attr_width;
+                        $height = $attr_height;
+                    } elseif ( isset( $viewbox_width, $viewbox_height ) ) {
+                        $width  = $viewbox_width;
+                        $height = $viewbox_height;
+                    }
                 } else {
-                    $width  = $viewbox_width;
-                    $height = $viewbox_height;
+                    if ( isset( $viewbox_width, $viewbox_height ) ) {
+                        $width  = $viewbox_width;
+                        $height = $viewbox_height;
+                    } elseif ( isset( $attr_width, $attr_height ) ) {
+                        $width  = $attr_width;
+                        $height = $attr_height;
+                    }
                 }
```

Comment:
This is awesome. I'm assuming we can't use php 7 stuff like the null coalescing operator to improve readability?

Reply:
I do believe our minimum PHP version is 7, so we could use the null coalescing operator here, though I guess I'm not sure where that would go? Happy to look at changing this if you have a suggestion on how this could be done better.